### PR TITLE
ci: Create automatic pr from patch

### PR DESCRIPTION
**Pull request by victorrabelozup**
## Description

Created a new action flow to create a new PR automatically whenever a patch(hotfix) is launched.
Solution: 
-  Create a new branch base in the recently generated tag
- Open a pull request from this branch to master

## Related Issues

#460 #580 

## Tests

Here is a link from an action that create the PR triggered from a release:
https://github.com/ZupIT/beagle/runs/930610713?check_suite_focus=true

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [DCO].
- [x] All existing and new tests are passing.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*

[issue database]: https://github.com/ZupIT/beagle/issues
[DCO]: https://developercertificate.org/

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+###  Workflow
+###
+platform :workflow do
+  desc "Open PR from patch"
+  lane :openPRFromPatch do |paramTag|
+    tagName = paramTag[:paramTag]
+    if !tagName.index("-")
+      puts "This release is not a hotfix."
+      puts "Exiting..."
+      next
```
Comment (matheusribeirozup) [inline fastlane/Fastfile:11]: Does the keyword `next` in this case work in the same way as the `else`?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+    branchName = tagName.split("/").last
+    sh("git checkout -b " + branchName)
+    push_to_git_remote(
+      tags: false,
+      set_upstream: true
+      )
+    create_pull_request(
+      api_token:paramTag[:token],
+      repo:"ZupIT/beagle",
+      title:"fix: Merge branchName patch with master",
```
Comment (uziasferreirazup) [inline fastlane/Fastfile:22]:
```suggestion
      title:"fix: Merge #{branchName} patch with master",
```

File: fastlane/Fastfile (hunk 1 above)
Comment (victorrabelozup) [reply fastlane/Fastfile:11]: it works like a `return`.

File: fastlane/Fastfile (hunk 2 above)
Comment (victorrabelozup) [reply fastlane/Fastfile:22]: sorry and thanks for the correction :)

**Comment by theffc** [comment]
about PR title, we should use "feat: " when we are adding a new feature to our users that will bump Minor version. Here, I think we could use "ci: " which would only change PATCH version and will imply what we are really doing.
For more information: https://www.conventionalcommits.org/en/v1.0.0/

File: .github/workflows/postRelease.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: Beagle Post Release
```
Comment (theffc) [inline .github/workflows/postRelease.yml:1]:
```suggestion
name: Post per platform Hotfix Release
```

File: fastlane/Fastfile [hunk 4]
```diff
@@ -1,3 +1,30 @@
...
+    end
+    branchName = tagName.split("/").last
+    sh("git checkout -b release/" + branchName + "-hotfix")
+    push_to_git_remote(
+      tags: false,
+      set_upstream: true
+    )
+    create_pull_request(
+      api_token:ENV["REPO_TOKEN"],
+      repo:"ZupIT/beagle",
```
Comment (LG95) [inline fastlane/Fastfile:21]: Get this from ENV, which can be filled using Actions available variables, see https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context.
```suggestion
      repo:ENV["REPO"],
```

Comment (victorrabelozup) [reply fastlane/Fastfile:21]: done added this in a separate commit to also create the env variable from the yml file.

**Pull request merged**